src/zope/file/browser.zcml

```diff
@@ -2,9 +2,12 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:browser="http://namespaces.zope.org/browser"
     xmlns:zcml="http://namespaces.zope.org/zcml"
+	zcml:condition="installed zope.browserpage"
```

Comment:
The indentation seems inconsistent here.

Reply:
Thanks for catching that. That's a tab. My editor configuration was messed up for ZCML files in that directory tree.